### PR TITLE
feat: Add DISABLE_REASON to Agents List

### PR DESCRIPTION
**Pull request by humberto-garza**
# Justification
We added an internal mechanism to "reserve" agents by adding a "user capability" to agents in AzDO called `DISABLE_REASON`.
This way we know if an agent has been reserved by someone or something.
Now we want to be able to see from the Agent Page if the agent was reserved by querying that capability.

Note - The DISABLE_REASON will only work `if (atNI)`

# Implementation
While filtering the agents in the Agent Page, query the agents' capabilities and if the capability `DISABLE_REASON` is set, append an HTML element with the information.gim

# Testing
image

### CODE REVIEW

File: src/azdo-pr-dashboard.user.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 
 // @name         More Awesome Azure DevOps (userscript)
-// @version      3.3.5
```
Comment (alejandro5042) [inline src/azdo-pr-dashboard.user.js:4]: Since this is a new feature, we should bump the minor version (the middle number). This project follows [semver](https://semver.org/).

File: src/azdo-pr-dashboard.user.js [hunk 2]
```diff
@@ -267,7 +275,7 @@
     });
 
     // Status of agents can change as the table is constantly updating.
-    setInterval(filterAgentsDebouncer(), 10000);
+    setInterval(filterAgentsDebouncer(), 30000);
```
Comment (alejandro5042) [inline src/azdo-pr-dashboard.user.js:278]: Why the change to 30s?

**Comment by alejandro5042** [comment]
This is set to work only at NI, right?

**Review by alejandro5042 — APPROVED** [review]

File: src/azdo-pr-dashboard.user.js (hunk 1 above)
Comment (humberto-garza) [reply src/azdo-pr-dashboard.user.js:4]: Done!

**Comment by humberto-garza** [comment]
> This is set to work only at NI, right?

@alejandro5042, it was not. But I added an `if (atNI)` to make it such that the DISABLE_REASON logic only works at NI.
